.editorconfig

```
@@ -0,0 +1,20 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space # useTabs in prettier
```

Maybe the boolean value could be added here for clarification?

```suggestion
indent_style = space # useTabs=false in prettier
```